homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
```

Comment:
Since you are creating more than one option, use the plural form `createOptions`.